### PR TITLE
v1.3.0 Release

### PR DESCRIPTION
**Pull request by james2791**
## Release checklist:
- [x] `CHANGELOG.md` updated
- [ ] `README.md` version examples updated
- [ ] `docs/` updated

Resolved Organization resource allowed API IP address persistance and pipeline schedule `enabled` setting (more for the disabling of a schedule) through its respective mutation

### CODE REVIEW

**Comment by james2791** [comment]
Docs and Readme always point to `v1.0+` nowadays, they most likely can be removed from the template

**Review by mcncl — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Changelog
 
 ## Unreleased
```
Comment (mcncl) [inline CHANGELOG.md:3]: I feel like we can remove `Unreleased` if there isn't anything in it.

Comment (james2791) [reply CHANGELOG.md]: Fresh 👍

**Review by lizrabuya — APPROVED** [review]
🚀

**Review by mcncl — APPROVED** [review]

**Pull request merged**